packages/astro/src/cli/info/index.ts

```diff
@@ -49,13 +49,19 @@ async function copyToClipboard(text: string) {
 	} else if (system === 'win32') {
 		command = 'clip';
 	} else {
-		// Unix: check if `xclip` is installed
-		const output = execSync('which xclip', { encoding: 'utf8' });
-		if (output[0] !== '/') {
-			// Did not find a path for xclip, bail out!
-			return;
+		try {
+			// Unix: check if `xclip` is installed
+			const output = execSync('which xclip', { encoding: 'utf8' });
+			if (output[0] !== '/') {
+				// Did not find a path for xclip, bail out!
+				return;
+			}
+			command = 'xclip -sel clipboard -l 1';
+		}
+		catch (e) {
+			// Did not find xclip, bail out!
+			return
```

Comment:
```suggestion
      console.error('Did not find xclip'); 
			return
```

Reply:
I think we just want to bail out than show console messages right, @matthewp?

Reply:
hey @dreyfus92, quick question, wouldn't this
```
console.error(colors.red(`\nSorry, something went wrong!`) + ` Did not find xclip.`);
console.error(e);
```
be better? and what about the `if output` block? I don't think it's needed, I guess only this is enough
```
execSync('which xclip', { encoding: 'utf8' });
command = 'xclip -sel clipboard -l 1';
```
but I'm not sure

Reply:
- the output block is required as in case execSync exists, there's no error and we have to evaluate if there's a path present for xclip.
- right now with error handling, the intention is just to bail out I believe

Reply:
@vasfvitor The simple return works, but the extra logging and error handling improves transparency.

Reply:
thanks for the explanation

Reply:
The error should be safe to ignore silently. I doubt many people know what xclip is to begin with, and if its absence is something to "fix", might confuse more to bring it up.